### PR TITLE
Updating KFMBlockOptIn policy

### PR DESCRIPTION
**Pull request by cartergreen**
KFMBlockOptIn Policy has been updated with a new reg key to redirect known folders back to user's device.

### CODE REVIEW

**Comment by cartergreen** [comment]
@adeejoseph FYI

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -323,14 +323,17 @@ This setting prevents users from moving their Documents, Pictures, and Desktop f
 > [!NOTE]
 > Moving known folders to personal OneDrive accounts is already blocked on domain-joined PCs.
   
-If you enable this setting, users aren't prompted with a window to protect their important folders, and the *Manage backup* command is disabled. If the user has already moved their known folders, the files in those folders remain in OneDrive. This setting does not take effect if you've enabled "Prompt users to move Windows known folders to OneDrive" or "Silently move Windows known folders to OneDrive."
+If you enable this setting, users aren't prompted with a window to protect their important folders, and the *Manage backup* command is disabled. If the user has already moved their known folders, the files in those folders will remain in OneDrive. To redirect the known folders back to user's device, please select "No." This setting does not take effect if you've enabled "Prompt users to move Windows known folders to OneDrive" or "Silently move Windows known folders to OneDrive."
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md:326]:
```suggestion
If you enable this setting, users aren't prompted with a window to protect their important folders, and the *Manage backup* command is disabled. If the user has already moved their known folders, the files in those folders will remain in OneDrive. To redirect the known folders back to the user's device, please select "No." This setting does not take effect if you've enabled "Prompt users to move Windows known folders to OneDrive" or "Silently move Windows known folders to OneDrive."
```

File: OneDrive/use-group-policy.md [hunk 2]
```diff
@@ -323,14 +323,17 @@ This setting prevents users from moving their Documents, Pictures, and Desktop f
...
   
 If you disable or do not configure this setting, users can choose to move their known folders.
   
 Enabling this policy sets the following registry key value to 1:
   
 `[HKLM\SOFTWARE\Policies\Microsoft\OneDrive]"KFMBlockOptIn"="dword:00000001"`
-  
+
+To redirect the known folders back to user's device and enable this policy set the following registry key value to 2:
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md:334]:
```suggestion
To redirect the known folders back to the user's device and enable this policy, set the following registry key value to 2:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**